app/assets/stylesheets/application.bootstrap.scss

```diff
@@ -11,6 +11,11 @@ $container-max-widths: (
 @import "bootstrap-icons/font/bootstrap-icons";
 @import "dropzone";
 
+// Stop the page from jumping when switching, e.g., to a tab containing a scrollbar
+html {
+  margin-left: calc(100vw - 100%);
+}
```

Comment:
You can see this on the work form, for instance, when switching from the title tab to the terms of deposit tab (or any other tab that renders a scrollbar).